araproc/analysis/snr.py

```diff
@@ -106,21 +106,22 @@ def get_avg_snr(wave_bundle, chans=None):
     ----------
     wave_bundle: dict of TGraphs
         Dictionary of waveform TGraphs to be averaged.
-    chans: list
-        List of dictionary keys to average over. If None,
-        averages over all keys in dictionary.
+    excluded_channels: list
+        List of dictionary keys to exclude from average.
 
     Returns
     -------
     avg_snr : float
         The average SNR.
     """
 
-    if(chans is None):
-      chans = list(wave_bundle.keys())
+    chans = list(wave_bundle.keys())
```

Comment:
I don't actually think you have to do this? Doesn't it by default iterate over the keys?

Reply:
Ya thats true. Could've cleaned it up a bit more. On the other hand this is probably a bit more explicit what is being done. I'll leave it in for now, can simplify later.

Reply:
Ok! I'll close this for now, it's a really minor nitpick.